### PR TITLE
fix(flux): return internal influxdb error for cursor conflict

### PR DESCRIPTION
**Pull request by jlapacik**

### CODE REVIEW

File: storage/flux/table.gen.go [hunk 1]
```diff
@@ -182,7 +182,13 @@ func (t *floatGroupTable) advanceCursor() bool {
 		if typedCur, ok := cur.(cursors.FloatArrayCursor); !ok {
 			// TODO(sgc): error or skip?
 			cur.Close()
-			t.err = errors.Errorf("expected float cursor type, got %T", cur)
+			t.err = &influxdb.Error{
+				Code: influxdb.EInternal,
```
Comment (jsternberg) [inline storage/flux/table.gen.go:186]: This will report itself as "An internal server error occurred" just like the previous error. Is this intentional?

Comment (jlapacik) [reply storage/flux/table.gen.go:186]: No that is not intentional. Is that because `Msg` is nil?

Comment (jsternberg) [reply storage/flux/table.gen.go:186]: It's because of the code. The `EInternal` code indicates it is an internal error that shouldn't be shown to users.

Comment (jlapacik) [reply storage/flux/table.gen.go:186]: Should it be `Invalid` instead?

Comment (jsternberg) [reply storage/flux/table.gen.go:186]: Yes, but probably try to make it a more informed error message like "attempted to combine rows of different types" or something. Then use the internal error as the nested error on the `influxdb.Error` type.

File: storage/flux/reader.go [hunk 2]
```diff
@@ -17,6 +17,32 @@ import (
...
+	case cursors.UnsignedArrayCursor:
+		got = "unsigned"
+	case cursors.StringArrayCursor:
+		got = "string"
+	case cursors.BooleanArrayCursor:
+		got = "boolean"
+	default:
+		got = "invalid"
+	}
+	return fmt.Sprintf("schema collision: cannot group %s and %s types together", err.typ, got)
```
Comment (jlapacik) [inline storage/flux/reader.go:43]: @jsternberg does this error message need more clarity?

Comment (jsternberg) [reply storage/flux/reader.go:43]: Oh I didn't notice this. This error message is great. Yes, just use `EInvalid` and you'll be fine.

Comment (jlapacik) [reply storage/flux/reader.go:43]: Done.

**Pull request merged**